### PR TITLE
Document known issues for v1.3.0

### PR DESCRIPTION
**Pull request by vishh**

### CODE REVIEW

**Comment by vishh** [comment]
cc @dchen1107 @lavalamp

File: CHANGELOG.md [hunk 1]
```diff
@@ -127,7 +127,25 @@ binary | sha1 hash | md5 hash
 
 ## Known Issues and Important Steps before Upgrading
 
-* *Instructions coming soon*
+The following versions of Docker Engine are supported - *v1.10*, *v1.11*
+Although *v1.9* is still compatible, we recommend upgrading to one of the supported versions.
+
+### kubernetes Core Known Issues
```
Comment (dchen1107) [inline CHANGELOG.md]: Should we provide the workaround here for each known issue?

File: CHANGELOG.md [hunk 2]
```diff
@@ -127,7 +127,25 @@ binary | sha1 hash | md5 hash
...
+
+- Kube Proxy crashes infrequently due to a docker bug ([#24000](https://github.com/docker/docker/issues/24000))
```
Comment (dchen1107) [inline CHANGELOG.md]: Workaround for this: restart docker

File: CHANGELOG.md [hunk 3]
```diff
@@ -127,7 +127,25 @@ binary | sha1 hash | md5 hash
...
+- CORS works only in insecure mode ([24086](https://github.com/kubernetes/kubernetes/issues/24086))
+- Persistent volume claims gets added incorrectly after being deleted under stress. Happens very infrequently. ([#26082](https://github.com/kubernetes/kubernetes/issues/26082))
+ 
+### Docker runtime Known Issues
+
+- Kernel crash with Aufs storage driver on Debian Jessie (#[27885](https://github.com/kubernetes/kubernetes/issues/27885))
```
Comment (dchen1107) [inline CHANGELOG.md]: Point the user to the admin / user guide for node problem detector? cc/ @Random-Liu

File: CHANGELOG.md [hunk 4]
```diff
@@ -127,7 +127,25 @@ binary | sha1 hash | md5 hash
...
+The following versions of Docker Engine are supported - *v1.10*, *v1.11*
+Although *v1.9* is still compatible, we recommend upgrading to one of the supported versions.
+
+### kubernetes Core Known Issues
+
+- Kube Proxy crashes infrequently due to a docker bug ([#24000](https://github.com/docker/docker/issues/24000))
+- CORS works only in insecure mode ([24086](https://github.com/kubernetes/kubernetes/issues/24086))
+- Persistent volume claims gets added incorrectly after being deleted under stress. Happens very infrequently. ([#26082](https://github.com/kubernetes/kubernetes/issues/26082))
+ 
+### Docker runtime Known Issues
```
Comment (dchen1107) [inline CHANGELOG.md]: File descriptor leakage issue was found for docker 1.11.X

https://github.com/docker/containerd/issues/275

File: CHANGELOG.md [hunk 5]
```diff
@@ -127,7 +127,25 @@ binary | sha1 hash | md5 hash
 
 ## Known Issues and Important Steps before Upgrading
 
-* *Instructions coming soon*
+The following versions of Docker Engine are supported - *v1.10*, *v1.11*
```
Comment (dchen1107) [inline CHANGELOG.md]: should we point to our validation issue for each release here, so some user can take a look at our validation results and issues if they wanted to know more detail?

File: CHANGELOG.md [hunk 6]
```diff
@@ -127,7 +127,25 @@ binary | sha1 hash | md5 hash
...
+Although *v1.9* is still compatible, we recommend upgrading to one of the supported versions.
```
Comment (dchen1107) [inline CHANGELOG.md]: Please also document: Kubelet 1.3 dropped the support for docker 1.8.X and below.

**Comment by dchen1107** [comment]
cc/ @matchstick

File: CHANGELOG.md (hunk 5 above)
Comment (vishh) [inline CHANGELOG.md]: Good idea. Adding links to the validation issues.

File: CHANGELOG.md (hunk 1 above)
Comment (vishh) [inline CHANGELOG.md]: Yeah good idea. Except for the first known issue, I can't find a workaround for the rest though.

File: CHANGELOG.md (hunk 4 above)
Comment (vishh) [inline CHANGELOG.md]: Cool. I added this and also the additional memory overhead one.

**Comment by vishh** [comment]
@dchen1107 Thanks for the review. PTAL

**Comment by dchen1107** [comment]
Content lgtm. But please let @erictune or @david-mcmahon take another look at the format? Thanks!

File: CHANGELOG.md [hunk 7]
```diff
@@ -122,12 +122,37 @@ binary | sha1 hash | md5 hash
...
   * Adding loadBalancer services and nodeports services to quota system
 
 ## Known Issues and Important Steps before Upgrading
 
-* *Instructions coming soon*
+The following versions of Docker Engine are supported - *[v1.10](https://github.com/kubernetes/kubernetes/issues/19720)*, *[v1.11](https://github.com/kubernetes/kubernetes/issues/23397)*
+Although *v1.9* is still compatible, we recommend upgrading to one of the supported versions.
+All prior versions of docker will not be supported.
+
+#### kubernetes Core Known Issues
```
Comment (david-mcmahon) [inline CHANGELOG.md]: Known issues is ##.  Why not make its sub headings ###.

Comment (vishh) [inline CHANGELOG.md]: The rendering was not great with `###`. I can switch it back if you think that's not an issue.

Comment (david-mcmahon) [inline CHANGELOG.md]: Ah, you tried both already.  SGTM.  LGTM.

**Comment by erictune** [comment]
LGTM

File: CHANGELOG.md (hunk 3 above)
Comment (Random-Liu) [inline CHANGELOG.md]: https://github.com/kubernetes/node-problem-detector/pull/22 is not merged yet. Will update the document after the PR gets merged. >"<

Comment (vishh) [inline CHANGELOG.md]: Thanks @Random-Liu

**Pull request merged**